frontend/scripts/componentSchemas/run.ts

```diff
@@ -31,15 +31,21 @@ const addProperties = (propertyKeys: string[]) => {
   allPropertyKeys.push(...propertyKeys);
 };
 
-const generateComponentSchema = (name: string, layoutSchema: any) => {
+const generateComponentSchema = (name: string, layoutSchema: any, version: string) => {
   const definitionName = `Comp${name}`;
   console.log('definitionName: ', definitionName);
   const componentSchema = layoutSchema.definitions[definitionName];
   let schema: any = {
-    $id: `https://altinncdn.no/schemas/json/component/${name}.schema.v1.json`,
+    $id: `https://altinncdn.no/schemas/json/component/${name}.schema.v1.json`, // These links respond with 404?
     $schema: layoutSchema.$schema,
   };
 
+  // The v4 schema has external definitions. This code block is needed to fetch v4 properties correctly.
+  const externalDefinitionName = definitionName + 'External';
+  if (version == 'v4' && layoutSchema.definitions[externalDefinitionName]?.allOf) {
+    componentSchema.allOf = layoutSchema.definitions[externalDefinitionName].allOf;
+  }
+
```

Comment:
Looks like this has been a issue since june 17, regarding to this commit. https://github.com/Altinn/altinn-cdn/commit/1263986482f14042bcd61f71a15b8e05442d3ad2#diff-8da0a5cc848d5b07e2560838726b42ee9c9bfd8df607ee82aba81fc8bcc4f8fe. 😊

Reply:
Interesting! I see that we added our latest component, PaymentDetails, on june 4th. So the timeline makes sense.